code/concepts/concepts.cpp

```diff
@@ -0,0 +1,47 @@
+#include <iostream>
+#include <list>
+#include <vector>
+
+/*
+
+In the code below, we provide two implementations of the function increment().
+The second one is optimized for the iterators which fulfill the requirements of
+the concept RandomAccessIterator.
+
+1. Uncomment and complete the concept RandomAccessIterator : check that the class IterT has some operator +=.
+2. Find an existing concept in the standard library, which can replace the previous user-defined concept.
+
+*/
+
+/*
+
+template< typename IterT>
```

Comment:
```suggestion
template< typename IterT >
```
or maybe `<typename IterT>`?

Reply:
You can try to run clang-format, but I don't know whether it will do well with the "TO BE COMPLETED"

Reply:
Do we have a clang-format file for the exercises? Because I also run clang-format in the past and noticed at some point that I used an uncommited `.clang-format` file, which was a left-over from the clang-format exercise :)

Reply:
Done. Committed soon.